Overseer/Services/Classes/UserService.cs

```diff
@@ -46,5 +49,18 @@ public User GetUserById(int id)
             if (user == null) throw new KeyNotFoundException("User not found");
             return user;
         }
+
+        private TokenDTO.RoleEnum GetEnumForRole(Role role)
```

Comment:
Tutaj nie da sie skorzystac z enuma wygenerowanego w bibliotece?
logika powstala jescze zanim to wszystko bylo ogarniete.